docs/docs/explanations/advanced-configuration.md

```diff
@@ -0,0 +1,815 @@
+---
+title: Advanced Configuration
```

Comment:
```suggestion
title: Advanced configuration
```

We've decided to [use sentence case](https://www.nebari.dev/docs/community/style-guide/#capitalization) for headings for consistency :)